templates/rustdoc/topbar.html

```diff
@@ -23,8 +23,8 @@
                     </li>
 
                     <li class="pure-menu-item">
-                        <a href="{{ crate_url | safe }}" class="pure-menu-link" class="description">
-                            {{ "cube" | fas(fw=true) }} {{ krate.description }}
+                        <a href="{{ crate_url | safe }}" class="pure-menu-link description" title="See {{ krate.name }} in docs.rs">
```

Comment:
What does the `description` class do here?

Reply:
Before we add two `class` attributes, so I simply merged them. :)